AWS_FOR_FLUENT_BIT_VERSION

```diff
@@ -1 +1 @@
-2.21.4
\ No newline at end of file
+2.21.5
```

Comment:
This is just a patch bump since the new fluent bit version contains only bug fixes